src/components/SafeImage/SafeImage.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { memo, VFC } from 'react';
+import { Box, BoxProps, Image, ImageProps } from '@chakra-ui/react';
+import { useSettingsContext } from '../../providers';
+
+interface SafeImageProps {
+
+  /** Whether the image is nsfw. */
+  readonly isNsfw?: boolean;
+
+  /** Wrapper container props. */
+  readonly containerProps?: BoxProps;
+
+  /** Image src. */
+  readonly src: string | null;
+}
+
+interface Props extends SafeImageProps, Omit<ImageProps, 'src'> {}
+
+/** Safe image. */
+const SafeImageComponent: VFC<Props> = ({ isNsfw = false, src, containerProps, ...rest }) => {
+  const { isNsfwContentAllowed } = useSettingsContext();
+
+  return (
+    <Box overflow="hidden" {...containerProps}>
+      <Image
+        src={src ?? undefined}
```

Comment:
Actually is there a reason to render image if `src` is empty string or  `null` without `alt` attribute?

Reply:
https://github.com/vndbre/vndbre/pull/122#discussion_r851698660

We would show fallback placeholder if there is no image.

Reply:
Just waiting till @moetazaneta implement universal placeholder for all our images.